composer.json

```diff
@@ -22,13 +22,13 @@
         "laminas/laminas-servicemanager": "<3.0"
     },
     "require": {
-        "php": "^7.3 || ~8.0.0",
-        "laminas/laminas-zendframework-bridge": "^1.0"
+        "php": "~7.4.0 || ~8.0.0 || ~8.1.0",
+        "laminas/laminas-zendframework-bridge": "^1.4.0"
```

Comment:
The bridge / custom autoloader can be removed here to avoid [performance problems](https://github.com/laminas/laminas-zendframework-bridge/issues/83). This includes another change:

```json
"replace": {
    "zendframework/zend-permissions-acl": "^3.2.0"
}
```
to:
```json
"conflict": {
    "zendframework/zend-permissions-acl": "*"
}
```